tests/api/test_dnf_module_package.py

```diff
@@ -0,0 +1,134 @@
+# -*- coding: utf-8 -*-
+
+
+from __future__ import absolute_import
+from __future__ import unicode_literals
+
+import os
+
+import dnf
+import libdnf
+
+from .common import TestCase
+
+
+class DnfModulePackageApiTest(TestCase):
+    def setUp(self):
+        self.base = dnf.Base()
+        repo = self.base.repos.add_new_repo(
+            'api-module-test-repo', self.base.conf,
+            baseurl=[os.path.join(os.path.dirname(__file__), "../modules/modules/_all/x86_64/")]
+        )
+        self.base.fill_sack(load_system_repo=False, load_available_repos=True)
+        moduleBase = dnf.module.module_base.ModuleBase(self.base)
+        modulePackages, nsvcap = moduleBase.get_modules('*')
+        self.modulePackage = modulePackages[0]
```

Comment:
Not really, it can only be created from `ModulePackageContainer` and there it loads the repo (it needs the data from modules.yaml which it gets from it).